classes/decision/types/traits/WithReviewAssignment.inc.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @file classes/decision/types/traits/WithReviewAssignment.inc.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class decision
+ *
+ * @brief Helper functions to provide review assignments related details associated with submission
+ */
+
+namespace PKP\decision\types\traits;
+
+use Exception;
+use PKP\db\DAORegistry;
+use PKP\decision\DecisionType;
+use PKP\submission\reviewAssignment\ReviewAssignment;
+use PKP\submission\reviewAssignment\ReviewAssignmentDAO;
+
+trait WithReviewAssignment
+{
+    /**
+     * Get all the review assignments based on review assignment states
+     *
+     * @param  int $submissionId            The targeted submission id
+     * @param  int $reviewRoundId           The targeted review round id
+     * @param  int $reviewAssignmentState   Review assignment state[active, completed, confirmed etc] based on reviews id get deduced
+     *
+     * @throws \Exception
+     *
+     * @return ReviewAssignment[]
+     *
+     */
+    protected function getReviewAssignments(int $submissionId, int $reviewRoundId, int $reviewAssignmentState): array
```

Comment:
`$reviewAssignmentState` -> `$status`. And in the docblock say:

> One of the `ReviewAssignment::REVIEW_ASSIGNMENT_STATUS_*` constants

Reply:
Same throughout this trait as well as in the `NotifyReviewers` trait.

Reply:
Done.